src/Bridge/Laravel/Resources/config/db-tools.php

```diff
@@ -0,0 +1,218 @@
+    | registered it into the container without using directly its class name.
+    |
+    | "default", null, or omitting the connection are equivalent and involve
+    | the use of the default implementation.
+    |
+    | To summarize, allowed values are:
+    |  - "default": alias of "datetime".
+    |  - "datetime": implementation producing names formatted as such:
+    |    <storage_directory>/YYYY/MM/<connection_name>-<datestamp>.<ext>.
+    |  - CLASS_NAME: class name of a custom strategy implementation.
+    |  - SERVICE_ID: identifier of a custom strategy implementation registered
+    |     as a service into the container.
+    |
+    */
+
+    'storage_filename_strategy' => env('DBTOOLS_STORAGE_FILENAME_STRATEGY', 'default'),
+
+    /*
+    | -------------------------------------------------------------------------
+    | Backup file expiration
+    | -------------------------------------------------------------------------
+    |
+    | Indicate when old backups can be considered obsolete.
+    |
+    | Use relative date/time formats:
+    | https://www.php.net/manual/en/datetime.formats.relative.php
+    |
+    */
+
+    'backup_expiration_age' => env('DBTOOLS_BACKUP_EXPIRATION_AGE', '3 months ago'),
+
+    /*
+    | -------------------------------------------------------------------------
+    | Backup excluded tables
+    | -------------------------------------------------------------------------
+    |
+    | List here database tables you don't want to include in your backups.
+    |
+    | If you have more than one database connection, it is strongly advised
+    | to configure this for each connection instead.
+    |
+    */
+
+    'backup_excluded_tables' => env('DBTOOLS_BACKUP_EXCLUDED_TABLES', []),
+
+    /*
+    | -------------------------------------------------------------------------
+    | Process timeouts
+    | -------------------------------------------------------------------------
+    |
+    | Default timeouts for backup and restoration processes.
+    |
+    */
+
+    'backup_timeout' => env('DBTOOLS_BACKUP_TIMEOUT', 600),
+    'restore_timeout' => env('DBTOOLS_RESTORE_TIMEOUT', 1800),
+
+    /*
+    | -------------------------------------------------------------------------
+    | Binaries & options
+    | -------------------------------------------------------------------------
+    |
+    | Specify here paths to backup and restoration binaries as well as their
+    | respective command line options.
+    |
+    | Warning: this will apply to all connections disregarding their database
+    | vendor. If you have more than one connection and if they use different
+    | database vendors or versions, please configure those for each connection
+    | instead.
+    |
+    | Default values depends upon vendor and are documented at
+    | https://dbtoolsbundle.readthedocs.io/en/stable/configuration.html
+    |
+    */
+
+    'backup_binary' => env('DBTOOLS_BACKUP_BINARY', '/usr/bin/pg_dump'),
```

Comment:
Are you setting defaults here? You should not, the default bin and options to use are directly injected by the adapter when empty: if you set values in the configuration, it will override it and possibly create bugs later.

Reply:
I asked myself about that point. I actually defined default values in two places while thinking that I will have to keep them in only one. But considering your other related comment, I probably have to remove all default values I precised and rely on the `Configuration` class.